protected/views/datasetSubmission/end.php

```diff
@@ -0,0 +1,27 @@
+<h2>The end</h2>
+<div class="clear"></div>
+
+<?php $this->renderPartial('_tabs_navigation', array('model' => $model)); ?>
+
+<div class="span12 form well">
+    <div class="form-horizontal">
+        <div id="end-grid" class="grid-view">
+            <p class="note">
+                All the dataset and sample metadata has been received, thank you.
+            </p>
+
+            <p class="note">
+                A curator will assigned to check the information and to advise you of how to supply the data files to us. Once the curator has checked the metadata they will provide you with a login for our private FTP server where you can upload the data files. Once uploaded you will be required to provide some file metadata, including a description, data type, and a file format for each file.
+            </p>
+
+            <p class="note">
+                If you do not hear from us within 2 working days please feel free to contact us by email <a href="mailto:support@gigasciencejournal.com">support@gigasciencejournal.com</a>, please include the GigaScience manuscript submission ID in the subject line.
```

Comment:
@only1chunts Shouldn't the email address be `database@gigasciencejournal.com` here? We don't have a `support@gigasciencejournal.com` anymore.

Reply:
Yes, please use database@gigasciencejournal.com, thanks.